ui/src/datasource/DatasourceOverview.vue

```diff
@@ -47,60 +40,74 @@
         />
 
         <template #[`item.trigger.interval`]="{ item }">
-          {{ getHoursFromMS(item.trigger.interval) }}h:{{ getMinutesFromMS(item.trigger.interval) }}m
+          {{ getFormattedInterval(item.trigger.interval) }}
+        </template>
+
+        <template #[`item.trigger.firstExecution`]="{ item }">
+          {{ getISODateString(item.trigger.firstExecution) }} {{ getISOTimeString(item.trigger.firstExecution) }}
+          <v-icon
+            v-if="isPastDate(item.trigger.firstExecution)"
+            small
+          >
+            mdi-check
+          </v-icon>
+          <v-icon
+            v-else
+            small
+          >
+            mdi-clock-outline
+          </v-icon>
         </template>
 
         <template #[`item.trigger.periodic`]="{ item }">
-          <v-switch
+          <v-simple-checkbox
             v-model="item.trigger.periodic"
-            class="ma-2"
             disabled
           />
         </template>
 
         <template #[`item.action`]="{ item }">
-          <v-btn
-            depressed
-            small
-            class="ma-2"
-            @click="onEdit(item)"
-          >
-            Edit
-            <v-icon
-              dark
-              right
-            >
-              mdi mdi-pencil
-            </v-icon>
-          </v-btn>
-          <v-btn
-            depressed
-            small
-            class="ma-2"
-            @click="onDelete(item)"
-          >
-            Delete
-            <v-icon
-              dark
-              right
-            >
-              mdi mdi-delete
-            </v-icon>
-          </v-btn>
-          <v-btn
-            depressed
-            small
-            class="ma-2"
-            @click="onCreatePipeline(item)"
-          >
-            Create Pipeline
-            <v-icon
-              dark
-              right
-            >
-              mdi mdi-pipe
-            </v-icon>
-          </v-btn>
+          <v-tooltip bottom>
```

Comment:
Very nice idea! Having it placed in the button seems like a waste of time.

Reply:
For some reason, the tooltips are displayed above the table when i remove the `bottom` directive. Looks very buggy, i'll investigate on this.

Reply:
Simply had to replace `bottom` with `top`